docs/reviewing-a-pull-request.md

```diff
@@ -0,0 +1,70 @@
+# Reviewing a Pull Request
+
+This guide is about reviewing pull requests on Exercism language tracks.
+
+## Philosophy
+
+* **Respond Kindly** We want every contributor to be glad they are here, even if we don't ultimately merge their pull request.
+* **Respond Quickly** Promptness encourages frequent, high-quality contributions from community members and other maintainers.
+* **Resolve Quickly** A pull request should either be actively in-progress, or it should be merged or closed.
+
+Closing pull requests kindly is not easy. Two great articles about this are:
+
+* [Kindly Closing Pull Requests](https://github.com/blog/2124-kindly-closing-pull-requests) by Mike McQuaid
+* [The Art of Closing](https://blog.jessfraz.com/post/the-art-of-closing/) by Jessie Frazelle
+
+## Reviewing an Exercism Exercise
+
+When reviewing a pull request, always remember the past. That is, remember that numerous users have already submitted an exercise and any significant change to an exercise invalidates their submissions to some extent.
+
+That said, we should not be afraid to change things for the better.
+
+### Consider the Structural Requirements
+
+We have a tool, [`configlet`](https://github.com/exercism/configlet#configlet), which does track-level linting. This runs on Travis CI when a pull request is submitted. You can also [download it](https://github.com/exercism/configlet/releases/latest) and run it locally.
+
+- Is the test file named correctly, per the guidelines in the language track?
```

Comment:
To me, placing this list right after the paragraph about configlet seems to imply that configlet checks these things (which I *think* isn't the case). A single sentence like "Other things you might like to check are" would help with this.